### PR TITLE
Javascript/javascript1/week2

### PR DESCRIPTION
**Pull request by adnan832**
My Week2 JavaScript1 homework

### CODE REVIEW

File: javascript/javascript1/week2/flightBooking.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+console.log (getFullname1("Adnan" , "Aziz"));
+const fullName1 = getFullname1("Abdul", "Hadi");
+const fullName2 = getFullname1("Abdul" , "Manan");
+console.log(fullName1);
+console.log(fullName2);
+
+//Second Part
+
+function getFullname(firstname, surname, useFormalName) {
+    if (firstname && surname == "")
```
Comment (florinmaticu) [inline javascript/javascript1/week2/flightBooking.js:17]: I think what you wanted to test is something similar to this.
```suggestion
    if ((firstname =="" )&& (surname == ""))
```

File: javascript/javascript1/week2/studentManager.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    if (typeof(studentName) == "string")
+   
+ {        
+    if (studentName == "Queen" || class07Students.length < 6) 
+    {
+        return class07Students.push(studentName);    
+        } 
+        
+        
+        else if (class07Students.includes(studentName)) {
```
Comment (florinmaticu) [inline javascript/javascript1/week2/studentManager.js:13]: I believe with your current _if_ logic you can still add two students with the same name. Can you try adding two students with the same name and see that ?

File: javascript/javascript1/week2/weatherWear.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+function weatherClothes(temp){
+    
+    if (temp<0){
+        return "Be ready for expected snow and wear heavy jackets and snow shoes";
+    }
+    else if (temp > 0 && temp <= 10){
```
Comment (florinmaticu) [inline javascript/javascript1/week2/weatherWear.js:6]: The current implementation is not correct if I call this function with _0_
```suggestion
    else if (temp >= 0 && temp <= 10){
```

File: javascript/javascript1/week2/flightBooking.js (hunk 1 above)
Comment (adnan832) [reply javascript/javascript1/week2/flightBooking.js:17]: don't you think it gives the same result i.e short form of writing the condition?

File: javascript/javascript1/week2/studentManager.js (hunk 2 above)
Comment (adnan832) [reply javascript/javascript1/week2/studentManager.js:13]: no i tried it doesnt allow same name

File: javascript/javascript1/week2/weatherWear.js (hunk 3 above)
Comment (adnan832) [reply javascript/javascript1/week2/weatherWear.js:6]: oh i see will change it right away

**Review by florinmaticu — APPROVED** [review]

**Pull request merged**